plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -199,6 +200,21 @@ default void acquireTableWriteLock(
         throw new UnsupportedOperationException();
     }
 
+    default long acquireTableExclusiveLock(
```

Comment:
The `long` returned value is aligned with HMS, but is not a good abtraction, i..e it's hard to implement for a different metastore.

Instead of `long`, let's return

```
interface MetastoreLock {
  void release();
}
```